setup.py

```diff
@@ -137,9 +137,16 @@ def rm(file):
         pass
 
 
-class Clean(clean):
+class Clean(Command):
+    user_options = []
+
+    def initialize_options(self):
+        pass
+
+    def finalize_options(self):
+        pass
+
     def run(self):
-        super().run()
         for source in cython_sources:
             genc = source.replace('.pyx', '.c')
```

Comment:
this runs our code when invoking `python setup.py clean` - but does it also run the setuptools-internal code that would run if we didn't override this?

Reply:
A: no, it does not.